.golangci.yml

```diff
@@ -0,0 +1,31 @@
+linters:
+  auto-fix: false
+  enable:
+    - nakedret 
+    - interfacer
+    - varcheck
+    - deadcode 
+    - structcheck 
+    - misspell
+    - maligned 
+    - ineffassign
+    - goconst
+    - goimports
+    - errcheck 
+    - dupl 
+    - unparam 
+    - golint 
+    - staticcheck
+    - unused
+    - gosimple
+    # missing to be added
+    # - lll (250), errcheck, gocyclo, gosec, scopelint, unconvert, stylecheck
```

Comment:
Hi @kensipe, 

Really thank you for your contribution.  Following some points.  

The goal of have `lll` is not allow the line to be bigger than the page (default value) which is bad since requires to use the scroll to check the code. However, was decided that it would be removed. I do not see the point to add it with 250 characters at all in this way. So, IMO we should not add the lll(250).

Also, see [here](https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/check-lint.sh#L47-L66) the current checks that has been used by the CI. In this way, the `errcheck` should be enabled as well. 

Then, it is missing ` --deadline 5m` as well.

Reply:
@camilamacedo86 
`--deadline`  has been deprecated for sometime... `timeout` is it's replacement.   
`timeout` is on line 88 of the makefile of this PR (with the same value as deadline previously had)

Reply:
`errcheck` was enabled on line 14 of this file.  It was mistakenly added to the line 22 comment... removed now.